### ISSUE TITLE
Encountering "Ignore Large Loss" Logs

### ISSUE THREAD

**Issue by Yuhuoo** [aim-uofa/PM-Loss#9]
Thank you for your excellent work. When I retrained the model, I noticed a large number of logs saying "ignore large loss" after the 5th epoch. Is this normal?

ignore large loss 0.35078105330467224 on step 54843
scene = ['dl3dv_b00f9830ced666d612465b88ead707a220206ff011a49a039b83797d729f14a4']
Warning: PM-Loss contains NaN values. Clamping to zero.
ignore large loss 0.39098137617111206 on step 54844
scene = ['dl3dv_deb6f09a43beef70b83788f11c5e2c8f1be5a11118e801fcd78832a0c9d606f7']
Warning: PM-Loss contains NaN values. Clamping to zero.
ignore large loss 0.4161478579044342 on step 54844
scene = ['dl3dv_888493063f9a557fadb95c6e22b38e854a4c1cb1bf184bc6ded51c19af1ea137']
Warning: PM-Loss contains NaN values. Clamping to zero.
Warning: PM-Loss contains NaN values. Clamping to zero.
ignore large loss 0.4163472354412079 on step 54845
scene = ['dl3dv_4ac4ff3af95c64ef2bdd00e80a9d535558c1d2b074bd9ced43bbb35354d56228']
ignore large loss 0.46288806200027466 on step 54845
scene = ['dl3dv_0055b63e80df6ec61d8793d6f42e4626038e88ea0e65000e439dd4241eff2570']
Warning: PM-Loss contains NaN values. Clamping to zero.
ignore large loss 0.46339040994644165 on step 54846
scene = ['dl3dv_656860f7cbc5e0b4d9381ed7ecc7cb030582d658c81a17067118962e06f70dd0']
Warning: PM-Loss contains NaN values. Clamping to zero.
Warning: PM-Loss contains NaN values. Clamping to zero.
ignore large loss 0.4152318835258484 on step 54847
scene = ['dl3dv_73da6f64372b8c3ac8f83780c205bcc9df1e63bead21bc6e4afcbd4e65b3f6c1']
Warning: PM-Loss contains NaN values. Clamping to zero.
ignore large loss 0.4418283700942993 on step 54847
scene = ['dl3dv_e96eb91b1d2d415aed129d78c987763a116cd8c69791184b38130f54305dbd9a']
Warning: PM-Loss contains NaN values. Clamping to zero.
ignore large loss 0.3981919586658478 on step 54848
scene = ['dl3dv_3c4af51254338f6f774e2b50edb42023e4ec1ec49c8ac1f69253d8bf4dda5179']
Warning: PM-Loss contains NaN values. Clamping to zero.
ignore large loss 0.39553922414779663 on step 54848
scene = ['dl3dv_b1301656dc59b500380b512595261e51fec3b24b479a3f6d2f2d1dbe94dd3c10']
Warning: PM-Loss contains NaN values. Clamping to zero.
ignore large loss 0.36930912733078003 on step 54849
scene = ['dl3dv_a4f637e4933867f2fbfc4c998b094e84065d07775e78eab2f2baa10a42e6cbee']
Warning: PM-Loss contains NaN values. Clamping to zero.
ignore large loss 0.3843815326690674 on step 54849
scene = ['dl3dv_f962479f0c2e21ab4c35ff9ff9b4c4fd3c363b89e67aeb701c7df70d7f38f6e8']

**Comment by dc-walker** [comment]
Hi @Yuhuoo ! Thanks for your interest in our work.

This phenomenon is likely because the DL3DV dataset contains a few "problematic" scenes (perhaps due to incorrect camera poses or other data anomalies). These scenes can cause incorrect gradients to backpropagate, which in turn leads to training instability (hence the large losses and NaN warnings).

You could try to observe which specific scene causes the training to collapse and then skip that scene during training. (There might be more than one problematic scene, so you may need to build a list of scenes to skip. If needed, you can email me, and I can provide the list I use.)

You only need to skip the specific scene that causes the training to crash; the subsequent "large loss" messages are likely just a result of the training having already become unstable.

Hope this helps!
